.git-blame-ignore-revs

```diff
@@ -16,4 +16,3 @@ be23fa0e422b51f4526828cb19b8105c89e5dcbb
 9094b639455aca34109f0c167f8efd12778d6ccf
 bd675709500ab32cdbd9e4c51dd9f0e169ad776c
 ff4b3c3948e6b616ce67916f6dc2b5de6583d0d2
-f2576837dcf1974f5b72d2cd0309ed7cafd48bc5
```

Comment:
Note that this is safe to remove as there is no commit with this hash in the history anyways. It shouldn't have been added in the first place as the hashes change on a squash-merge[^1].

[^1]: Due to the squash-merging, if we want to modify this file we have to merge the PR with the changes first, then modify `.git-blame-ignore-revs` in a follow-up PR.